tests/browser/batch_gift_entry/bge_custom_batch.robot

```diff
@@ -21,19 +21,18 @@ BGE Batch With Custom Values
     ...                       Expected Count of Gifts=2
     ...                       Expected Total Batch Amount=100
     Click BGE Button        Next
-    Select Duellist Values    Opportunity    Available Fields    Donation Name
-    Click Duellist Button    Opportunity    Move selection to Selected Fields
-    ${xpath}    Get NPSP Locator    bge.duellist    Payment    Selected Fields    Check/Reference Number
-    Execute JavaScript    window.document.evaluate('${xpath}', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.scrollIntoView(true)
-    Select Duellist Values    Payment    Selected Fields    Check/Reference Number
-    Click Duellist Button    Payment    Move selection to Available Fields
+    Click Element With Locator    bge.duellist    Opportunity    Available Fields    Donation Name
```

Comment:
I like this block of changes. While it might have made the test slightly easier to read with some highly specialized keywords (`select duellist values`, `click duellist button`, etc), I think having one general purpose keyword instead of multiple special-purpose keywords will help make it easier to learn and use the NPSP keywords.